### ISSUE TITLE
ThemeProvider support

### ISSUE THREAD

**Issue by jwlawrence** [Luinova/ember-styled-components#1]
It appears that this doesn't provide support for wrapping styled components with a theme provider component provided by styled-components, is that correct?

Great work on this, works very well so far. Let me know if you need help with anything.

**Comment by semanticpixel** [comment]
Yeah I am still working on some of the features, I will look into the ThemeProvider. 

And any help is greatly appreciated 😃
